service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/ClonePolicyAttributesStep.java

```diff
@@ -41,16 +49,44 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
             .getInputParameters()
             .get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
 
-    Optional<ApiTpsPaoGetResult> workspacePao =
+    Optional<ApiTpsPaoGetResult> sourcePao =
         tpsApiDispatch.getPaoIfExists(
             new BearerToken(userRequest.getRequiredToken()), sourceWorkspaceId);
 
-    if (workspacePao.isPresent()) {
+    if (!sourcePao.isPresent()) {
```

Comment:
So is the plan to backfill paos onto old workspaces at some point?

If so, then add a TODO

// Delete this block when all workspaces have Paos

If not, would it be cleaner to:

- Don't create source pao and don't call clonePao. Just call createPao on destination.